data/pb/BPA/DocTypeMapping.json

```diff
@@ -0,0 +1,24 @@
+{
+    "tenantId": "pb",
+    "moduleName": "BPA",
+    "DocTypeMapping": [
+        {
+            "applicationType": "BUILDING_PLAN_SCRUTINY",
+            "ServiceType": "NEW_CONSTRUCTION",
+            "RiskType": "High",
+            "docTypes": [
+                {
+                    "code": "VOTERID",
+                    "required": true,
+                    "sequence":0
```

Comment:
Why do we need sequence in master data? Can we use the order of the array it self.